documentation/docs/mock-apps/services/get-value-from-local-storage.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 28
```

Comment:
Is it a duplicate? If it's not, skip this comment

<img width="349" alt="image" src="https://github.com/user-attachments/assets/a1080acf-011b-422d-9d39-f905dc0648f6">

Reply:
Yep, it’s not duplicated in the `Services` tab. I think we only need to be concerned when using the same `sidebar_position` within the same parent tab.